diff --git a/src/openslide-vendor-ventana.c b/src/openslide-vendor-ventana.c
index 80f5b22..f176489 100644
--- a/src/openslide-vendor-ventana.c
+++ b/src/openslide-vendor-ventana.c
@@ -68,6 +68,7 @@ static const char ATTR_OVERLAP_X[] = "OverlapX";
 static const char ATTR_OVERLAP_Y[] = "OverlapY";
 static const char DIRECTION_RIGHT[] = "RIGHT";
 static const char DIRECTION_UP[] = "UP";
+static const char DIRECTION_LEFT[] = "LEFT";  // VarunaPoC patch
 
 #define PARSE_INT_ATTRIBUTE_OR_RETURN(NODE, NAME, OUT, RET)	\
   do {								\
@@ -569,6 +570,14 @@ static struct bif *parse_level0_xml(const char *xml,
           &area->tiles[tile2_row * area->tiles_across + tile2_col];
         joint = &tile->left;
         ok = (tile2_col == tile1_col + 1 && tile2_row == tile1_row);
+      } else if (!xmlStrcmp(direction, BAD_CAST DIRECTION_LEFT)) {
+        // VarunaPoC patch: treat LEFT as RIGHT
+        // Empirical testing shows identical images
+        // See: https://github.com/openslide/openslide/issues/234
+        struct tile *tile =
+          &area->tiles[tile2_row * area->tiles_across + tile2_col];
+        joint = &tile->left;
+        ok = (tile2_col == tile1_col + 1 && tile2_row == tile1_row);
       } else if (!xmlStrcmp(direction, BAD_CAST DIRECTION_UP)) {
         // get top joint of bottom tile
         struct tile *tile =
